cloudnet-plugins/cloudnet-simplenametags/src/main/java/de/dytanic/cloudnet/ext/simplenametags/CloudNetSimpleNameTagsPlugin.java

```diff
@@ -21,4 +45,121 @@ public void onDisable() {
         CloudNetDriver.getInstance().getEventManager().unregisterListeners(this.getClass().getClassLoader());
         Wrapper.getInstance().unregisterPacketListenersByClassLoader(this.getClass().getClassLoader());
     }
+
+    public void updateNameTags(Player player) {
+        updateNameTags(player, null);
+    }
+
+    public void updateNameTags(Player player, Function<Player, IPermissionGroup> playerIPermissionGroupFunction) {
+        updateNameTags(player, playerIPermissionGroupFunction, null);
+    }
+
+    public void updateNameTags(Player player, Function<Player, IPermissionGroup> playerIPermissionGroupFunction,
+                               Function<Player, IPermissionGroup> allOtherPlayerPermissionGroupFunction) {
+        Preconditions.checkNotNull(player);
+
+        IPermissionUser playerPermissionUser = CloudNetDriver.getInstance().getPermissionManagement().getUser(player.getUniqueId());
+        AtomicReference<IPermissionGroup> playerPermissionGroup = new AtomicReference<>(playerIPermissionGroupFunction != null ? playerIPermissionGroupFunction.apply(player) : null);
+
+        if (playerPermissionUser != null && playerPermissionGroup.get() == null) {
+            playerPermissionGroup.set(CloudNetDriver.getInstance().getPermissionManagement().getHighestPermissionGroup(playerPermissionUser));
+
+            if (playerPermissionGroup.get() == null) {
+                playerPermissionGroup.set(CloudNetDriver.getInstance().getPermissionManagement().getDefaultPermissionGroup());
+            }
+        }
+
+        int sortIdLength = CloudPermissionsManagement.getInstance().getGroups().stream()
```

Comment:
The CloudPermissionManagement is deprecated, we should use CloudNetDriver#getPermissionManagement